web_external/User/InviteUserView.js

```diff
@@ -2,28 +2,40 @@ isic.views.InviteUserView = isic.View.extend({
     events: {
         'submit #isic-user-invite-form': function (event) {
             event.preventDefault();
-            // this.$('#isic-user-invite-submit').prop('disabled', true);
+            this.$('#isic-user-invite-submit').prop('disabled', true);
+
+            var data = {
+                login: this.$('#isic-user-invite-new-login').val(),
+                email: this.$('#isic-user-invite-new-email').val(),
+                firstName: this.$('#isic-user-invite-new-firstname').val(),
+                lastName: this.$('#isic-user-invite-new-lastname').val()
+            };
+
+            var validityPeriod = this.$('#isic-user-invite-period').val();
+            if ($.trim(validityPeriod).length > 0) {
```

Comment:
ES5 has [`String.prototype.Trim`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/Trim). Why not use that?